index.bs

```diff
@@ -3589,11 +3634,13 @@ throughout the rest of this standard.
 </emu-alg>
 
 <h4 id="enqueue-value-with-size" aoid="EnqueueValueWithSize" throws>EnqueueValueWithSize ( <var>queue</var>,
-<var>value</var>, <var>size</var> )</h4>
+<var>value</var>, <var>size</var>, <var>targetRealm</var> )</h4>
```

Comment:
Oh, right, I also had a preference for throwing the `RangeError` for invalid strategy return values before potentially throwing the `DataCloneError`.